scrapers/scrape_bs.py

```diff
@@ -125,4 +125,7 @@
     sc.find(r'Todesfälle\s*im\s*Kanton\s*Basel-Stadt\s*beträgt(?:\s*\S+)?\s*insgesamt\s*([0-9]+)\b', d) or \
     sc.find(r'Die\s*Zahl\s*der\s*Todesfälle\s*im\s*Kanton\s*Basel-Stadt\s*beträgt\s*.*unverändert\s*([0-9]+)\b', d)
 
+dd.isolated = sc.int_or_word(sc.find(r'\s+(\S+)\s+aktiven\s+Fällen\s+in\s+Isolation', d))
```

Comment:
@dominikgehl the text today is a bit different

> Die Zahl der 979 Infektionen setzt sich zusammen aus 923 genesenen Personen (-), 50 Todesfällen (-) und sechs aktiven Fällen (+ 1).

So for the `current_isolated` the part "in Isolation" is optional

Reply:
updated